codice/ddf
# DDF-1960 merged catalog and content frameworks

#### What does this PR do?
Removes the DDF Content application and merges the functionality into DDF Catalog
#### Who is reviewing it?
@coyotesqrl @lessarderic @pklinef @kcwire @jlcsmith @Lambeaux @brendan-hofmann 
#### How should this be tested?
run through all itests
start up DDF, ingest a file using the Search UI ingest, perform a query and see that the result exists, download the product and see that the files match
ingest a metacard XML document by posting the document as the body of the message, see that a metacard result exists for this document and that no product is available
#### Any background context you want to provide?
Additional I-tests will be added in a separate PR as there were none existing for most of DDF Content
#### What are the relevant tickets?
DDF-1960
#### Screenshots (if appropriate)
#### Checklist:

 - removed content app
 - rewrote file system storage provider to temporarily store products until commit
 - updated CatalogFramework API with new create and update methods
 - updated Search UI to point to catalog REST endpoint
 - updated catalog REST endpoint to use new create and update methods for multipart requests
